player/configfiles.c

```diff
@@ -383,9 +380,30 @@ void mp_delete_watch_later_conf(struct MPContext *mpctx, const char *file)
     }
 
     char *fname = mp_get_playback_resume_config_filename(mpctx, file);
-    if (fname)
+    if (fname) {
         unlink(fname);
-    talloc_free(fname);
+        talloc_free(fname);
+    }
+
+    if (mp_is_url(bstr0(fname)))
```

Comment:
I got a segfault from this line. Seems like `fname` can was freed a few lines up?

Reply:
I was using the wrong variable. I wonder why I never got a segfault.

Reply:
Ah yeah there we go.